File: easy/oddCells/oddCells.java
import java.util.Arrays;

class Solution {
  public int oddCells(int n, int m, int[][] indices) {
    int N = 0;
    int[][] arr = new int[][]{};
    for (int i = 0; i < n; i++) {
      for (int j = 0; j < m; j++) {
        arr[i][j] = 10;
      }
    }
    System.out.println(Arrays.toString(arr));
    return N;
  }

  public static void main(String[] args) {
    Solution solution = new Solution();
    int n = 2, m = 3;
    int[][] indices = new int[][] { { 1, 1 }, { 0, 0 } };
    int N = solution.oddCells(n, m, indices);
    System.out.println(N);
  }
}

File: easy/oddCells/oddCells.go
package main

import "fmt"

func oddCells(n int, m int, indices [2][2]int) int {
	var N int
	arr := make([][]int, n, m)
	fmt.Println(arr)
	for i := 0; i < n; i++ {
		for j := 0; j < m; j++ {
			fmt.Println(i)
			fmt.Println(j)
			arr[i][j] = 10
			fmt.Println(arr)
		}
	}
	fmt.Println(arr)
	return N
}

func main() {
	indices := [2][2]int{
		{0, 1},
		{1, 1}}
	N := oddCells(2, 3, indices)
	fmt.Println(N)
}
